tests/integration/routes/account/patch-account-test.js

```diff
@@ -45,10 +45,14 @@ var routeOptions = {
   }
 }
 function responseMock () {
-  // user document gets fetched twice, once for session.find(),
-  // then for account.update(). We might improve that by implementing
+  // user document gets fetched 3 times:
+  // 1) session.find()
+  // 2) account.update()
+  // 3) session.add()
```

Comment:
`session.add()` does not fetch the user document, otherwise we need to run `mockUserFound()` 3 times below for it to work. I think we can remove that comment line?